Keas.Mvc/ClientApp/components/Keys/KeySerialTable.tsx

```diff
@@ -16,142 +20,184 @@ interface IProps {
   onEdit?: (keySerial: IKeySerial) => void;
 }
 
-interface IFilter {
-  id: string;
-  value: any;
+interface IFilterOption {
+  value: string;
+  displayText: string;
 }
 
-interface IRow {
-  original: IKeySerial;
+// UI for Key status column filter
+const KeyStatusColumnFilter = ({
+  column: { filterValue, setFilter }
+}) => {
+  // Render a multi-select box
+  return (
+    <select
+      value={filterValue}
+      style={{ width: '100%' }}
+      onChange={e => {
+        setFilter(e.target.value || undefined);
+      }}
+    >
+      {ReactTableStatusUtil.defaultFilterOptions.map(option => (
```

Comment:
@wongband feels here like the ReactTableStatusUtil is a bit overkill and it would be easier to read if you just wrote out the `<option></option>` in the select.  It would be a lot less lines

Also it looks like you are duplicating the filter.  I see two different `<select>`, here and in the static class.  And also two different filters with basically the same logic.  Did you just keep around the old stuff that isn't used anymore?